index.php

```diff
@@ -1,14 +1,21 @@
 <?php
-
 /**
  * Plugin Name: Gutenberg
  * Plugin URI: https://wordpress.github.io/gutenberg/
  * Description: Prototyping since 1440. Development plugin for the editor focus in core.
  * Version: 0.1
+ *
+ * @package WordPress
  */
 
+// Hook.
 add_action( 'admin_menu', 'gutenberg_menu' );
 
+/**
+ * Gutenberg's Menu.
+ *
+ * @since  4.8.0
+ */
```

Comment:
This block should describe (even if just in a short sentence) what the function actually *does*.

> Adds a new menu page for the Gutenberg editor.